Makefile

```diff
@@ -1,34 +1,121 @@
-OPERATOR_SDK_VERSION ?= v0.18.2
-IMAGE_REGISTRY ?= quay.io/oyahud
-OPERATOR_IMAGE ?= kubevirt-ssp-operator
-IMAGE_TAG ?= latest
-OPERATOR_BUILD_ARGS ?= "-v -mod=vendor"
+# Current Operator version
+VERSION ?= 0.0.1
```

Comment:
Shouldn't this be `1.0.0`?

Reply:
`1.0.0` is usually the first complete released version. This `0.0.1` was auto-generated. I think for now it is good, and later we can update it when the operator is released.